bin/install-theme.js

```diff
@@ -0,0 +1,29 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const inquirer = require('inquirer');
+const childProcess = require('child_process');
+
+function promptUserForTheme() {
+  return inquirer.prompt([
+    {
+      type: 'input',
+      name: 'theme',
+      message: 'What @edx/brand package do you want to install?',
+      default: '',
```

Comment:
nit: Should this default `@edx/brand-openedx@latest`? Or a required field? Worth a sanity check.